doc/coq.rst

```diff
@@ -0,0 +1,56 @@
+.. _coq-main:
+
+******
+Coq
+******
+
+Dune is also able to build Coq developments. A Coq project is a mix of
+Coq ``.v`` files and OCaml libraries, known as *Coq plugins*. To enable
+Coq support in a dune project, the language version should be selected
+in the ``dune-project`` file. For example:
+
+.. code:: scheme
+
+    (using coq 0.2)
+
+This will enable support for the ``coqlib`` stanza in the current project. If the
+language version is absent, dune will automatically add this line with the
+latest Coq version to the project file once a ``(coqlib ...)`` stanza is used anywhere.
+
+
+Basic Usage
+===========
+
+The basic form for defining Coq libraries is very similar to the OCaml form:
+
+.. code:: scheme
+
+    (coqlib
+     (name <module_prefix>)
+     (public_name <package.module_prefix>)
+     (synopsis <text>)
+     (modules <ordered_set_expr>)
+     (libraries <ocaml_libraries>)
+     (flags <coq_flags>))
+
+The stanza will build all `.v` files on the given directory, *à la*
+``coq_makefile``. The semantics of fields is:
```

Comment:
I would remove this first sentence entirely. It doesn't bring much information, refers to a tool that people may not know. And anyways, the rule is similar to what Dune would do for OCaml libraries: build all the modules in a directory unless `(modules ` is given a restricted list.

Reply:
I agree, however the idea was also to provide some information to users migrating to Dune form Coq makefile, will reword.